src/elements/OcMenuItem.vue

```diff
@@ -29,6 +27,12 @@ export default {
       default: "",
     },
   },
+  computed: {
+    $_ocMenuItem_menuItemClass() {
+      let classes = ["oc-menu-item"]
+      return classes
+    },
+  },
   methods: {
     onClick(val) {
```

Comment:
please apply naming convention to onClick - THX

Reply:
Fixed